apex/transformer/functional/fused_rope.py

```diff
@@ -78,8 +174,10 @@ def fused_apply_rotary_pos_emb(
     Returns:
         Tensor: The input tensor after applying RoPE
     """
-    return FusedRoPEFunc.apply(t, freqs, transpose_output_memory)
-
+    if not AITER_ROPE_BACKEND:
```

Comment:
If `AITER_ROPE_BACKEND` is not a runtime switchable option. I would add
```
FusedRoPEFunc = FusedRoPEFuncAiter if AITER_ROPE_BACKEND else FusedRoPEFuncApex
```
at the beginning of the file, and then replace this pattern with
```
return FusedRoPEFunc.apply(t, freqs, transpose_output_memory)
```
elsewhere.